test/utils/creds.go

```diff
@@ -1,23 +1,69 @@
 package utils
 
 import (
-	"os"
+	"fmt"
 
-	"github.com/spyzhov/ajson"
+	"github.com/amp-labs/connectors/utils"
 )
 
-// Credentials returns the credentials from the creds.json file.
-func Credentials(defaultFileName string) (*ajson.Node, error) {
-	fileName := defaultFileName
-
-	if fn, ok := os.LookupEnv("CREDENTIALS_FILE"); ok {
-		fileName = fn
+func AmpersandConnectionSchemaReaders(filePath string) []utils.Reader {
+	schema := []utils.Reader{
+		&utils.JSONReader{
+			FilePath: filePath,
+			JSONPath: "$.AccessToken.Token",
+			CredKey:  utils.AccessToken,
+		},
+		&utils.JSONReader{
+			FilePath: filePath,
+			JSONPath: "$.RefreshToken.Token",
+			CredKey:  utils.RefreshToken,
+		},
+		&utils.JSONReader{
+			FilePath: filePath,
+			JSONPath: "$.providerApp.clientId",
+			CredKey:  utils.ClientId,
+		},
+		&utils.JSONReader{
+			FilePath: filePath,
+			JSONPath: "$.providerApp.clientSecret",
+			CredKey:  utils.ClientSecret,
+		},
+		&utils.JSONReader{
+			FilePath: filePath,
+			JSONPath: "$.providerWorkspaceRef",
+			CredKey:  utils.WorkspaceRef,
+		},
+		&utils.JSONReader{
+			FilePath: filePath,
+			JSONPath: "$.providerApp.provider",
+			CredKey:  utils.Provider,
+		},
 	}
 
-	byteValue, err := os.ReadFile(fileName)
-	if err != nil {
-		return nil, err
-	}
+	return schema
+}
 
-	return ajson.Unmarshal(byteValue)
+func EnvVarSchemaReaders(prefix string) []utils.Reader {
```

Comment:
the word 'Schema' is not really necessary here, `EnvVarsReaders` would be a shorter function name.